src/compiler/reviews.md

```diff
@@ -27,6 +27,85 @@ delegate+` or `@bors delegate=username`. This will allow the PR author
+added.
+
+A "meaningful regression" in this case is up to the judgment of the person
+approving the revert. As a rule of thumb, this would be a bug in a stable
+or otherwise important feature that causes code to stop compiling, changes
+runtime behavior, or triggers a (warn-by-default or higher) lint incorrectly in
+real-world code.
+
+When these criteria are in doubt, and especially if real-world code is affected,
+revert the PR. This allows bleeding edge users to continue to use and report
+bugs on HEAD with a higher degree of certainty about where new bugs are introduced.
+
+Before being reverted, a PR should be shown to cause a regression with a fairly
+high degree of certainty (e.g. bisection on commits, or bisection on nightlies
+with one or more compiler team members pointing to this PR, or it's simply
+obvious to everyone involved). Only revert with lower certainty if the issue is
+particularly critical or urgent to fix.
+
+### Creating reverts
+
+The easiest method for creating a revert is to use the "Revert" button on
+Github. This appears next to the "bors merged commit abcd" message on a pull
+request, and creates a new pull request.
+
+![Location of the "Revert" button](revert-button.png)
+
+Alternatively, a revert commit can be created using the git CLI and then
+uploaded as a pull request:
+
+```terminal
+$ git revert -m 1 62d5bee
+```
+
+It's polite to tag the author and reviewer of the original PR so they know
+what's going on. You can use the following message template:
+
+```
+Reverts rust-lang/rust#123456
+cc @author @reviewer
+
+This revert is based on the following report of a regression caused by this PR:
+<link to issue or comment(s)>
+
+In accordance with the compiler team [revert policy], PRs that cause meaningful
+regressions should be reverted and re-landed once the regression has been fixed
+(and a regression test has been added, where appropriate).
+[revert policy]: https://forge.rust-lang.org/compiler/reviews.html#reverts
+
+Fear not! Regressions happen. Please rest assured that this does not
+represent a negative judgment of your contribution or ability to contribute
+positively to Rust in the future. We simply want to prioritize keeping existing
+use cases working, and keep the compiler more stable for everyone.
+
+r? compiler
+```
+
+If you have r+ privileges, you can self-approve a revert.
+
+Generally speaking, reverts should have elevated priority and match the `rollup`
+status of the PR they are reverting. If a non-rollup PR is shown to have no
+impact on performance, it can be marked `rollup=always`.
+
+### Forward fixes
+
+Often it is tempting to address a regression by posting a follow-up PR that,
+rather than reverting the regressing PR, instead augments the original in
+small ways without reverting its changes overall. However,
+if real-world users have reported being affected, this practice is strongly
+discouraged unless a high-confidence fix is already in the bors queue.
```

Comment:
I think this paragraph is fine as an explanation to contributors why we reach for `git revert` to resolve regressions. However, as a policy, in practice what we seem to most often do is assess whether a revert or a smaller targeted fix (or a smaller revert such as just flipping a stable -> unstable feature flag somewhere) is the safer option, especially when backporting changes to stable or beta branches. I think even then, we often lean toward reverting by default but most of our discussions really come down to "what is the least impacting change we can backport?".

Just wanted to capture that nuance. If this is more targeted at contributors and not intended to establish official team policy, feel free to ignore 🙂

Reply:
Backports are a useful case to consider. I updated the wording to try to address this, can you take a look?